es6/tests/e2e/modules.js

```diff
@@ -502,13 +502,17 @@ describe("Module Matcher API", function () {
 				},
 			};
 		}
-		this.name = "tag-example.docx";
-		this.options = {
-			modules: [module1()],
-		};
-		this.data = { first_name: "John" };
-		expect(this.renderV4().getFullText()).to.equal("l28!! John");
+		expect(
+			this.renderV4({
```

Comment:
Please use the API with `expectedText: "...."`.

Reply:
Done.